### PR TITLE
feat(doc): add section on homebrew dependencies and use jsonc syntax highlighting

### PR DESCRIPTION
**Pull request by revilowaldow**

### CODE REVIEW

**Review by orcnog — APPROVED** [review]
:)

File: _doc/Spec_Copy.md [hunk 1]
```diff
@@ -2,9 +2,70 @@
...
+
+`"includes"` is structured similarly to `"dependencies"` and enables the site to load additional content from other homebrew files that are stored in the homebrew repo.
+
+### Example
+
+An example homebrew file might contain the following information in it's `_meta` property:
+
+```jsonc
+"dependencies": {
+    "monster": [    //Use the property name first
```
Comment (TheGiddyLimit) [inline _doc/Spec_Copy.md:42]:
```suggestion
    "monster": [    // Use the property name first
```

spaces after yer `//`s

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
See comment

File: _doc/Spec_Copy.md (hunk 1 above)
Comment (revilowaldow) [reply _doc/Spec_Copy.md:42]: Yessir will fix tomorrow when PC is on 👍

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**